classes/security/Validation.php

```diff
@@ -484,87 +485,116 @@ public static function canAdminister($administeredUserId, $administratorUserId)
      *
      * @return int The authorized administration level
      */
-    public static function getAdministrationLevel(int $administeredUserId, int $administratorUserId, ?int $contextId = null): int
-    {
-        // You can administer yourself
-        if ($administeredUserId == $administratorUserId) {
+    public static function getAdministrationLevel(
```

Comment:
Can you check if indent on empty lines in your environment is the same as in the project?